From 4c9d5e107dd40f3a70c75d60b9e353109c3604f4 Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Fri, 16 Dec 2016 15:20:05 +0900
Subject: [PATCH 1706/2066] Revert "ASoC: rsnd: Support S2RAM"

This reverts commit 4a997f1ecdd5 ("ASoC: rsnd: Support S2RAM")

This implementation was not so good for upstream policy.
Since another implementation is applied in upstream,
this patch will be reverted here.

Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Signed-off-by: Hiroyuki Yokoyama <hiroyuki.yokoyama.vx@renesas.com>
---
 sound/soc/sh/rcar/core.c |   28 ----------------------------
 1 files changed, 0 insertions(+), 28 deletions(-)

diff --git a/sound/soc/sh/rcar/core.c b/sound/soc/sh/rcar/core.c
index ef9f199..9840ad9 100644
--- a/sound/soc/sh/rcar/core.c
+++ b/sound/soc/sh/rcar/core.c
@@ -1254,37 +1254,9 @@ static int rsnd_remove(struct platform_device *pdev)
 	return ret;
 }
 
-#ifdef CONFIG_PM_SLEEP
-static int rsnd_suspend(struct device *dev)
-{
-	struct rsnd_priv *priv = dev_get_drvdata(dev);
-
-	rsnd_adg_remove(priv);
-
-	return 0;
-}
-
-static int rsnd_resume(struct device *dev)
-{
-	struct rsnd_priv *priv = dev_get_drvdata(dev);
-	int ret;
-
-	ret = rsnd_adg_probe(priv);
-
-	return ret;
-}
-
-static SIMPLE_DEV_PM_OPS(rsnd_pm_ops,
-			rsnd_suspend, rsnd_resume);
-#define DEV_PM_OPS (&rsnd_pm_ops)
-#else
-#define DEV_PM_OPS NULL
-#endif  /* CONFIG_PM_SLEEP */
-
 static struct platform_driver rsnd_driver = {
 	.driver	= {
 		.name	= "rcar_sound",
-		.pm	= DEV_PM_OPS,
 		.of_match_table = rsnd_of_match,
 	},
 	.probe		= rsnd_probe,
-- 
1.7.5.4

